index.js

```diff
@@ -1,569 +0,0 @@
-function enterState(newState) {
-    let validTransitions = states[currentState].canChangeTo;
-    if (validTransitions.includes(newState)) {
-        currentState = newState;
-        console.log(roomLookUpTable[currentState].desc);
-    } else {
-        throw `Invalid state transition attempted from ${currentState} to ${newState}.`;
-    }
-}
-
-const roomLookUpTable = {
-    'bedRoom': bedRoom,
-    'downStairs': downStairs,
-    'palletTown': palletTown,
-    'rivalHouse': rivalHouse,
-    'routeOne': routeOne,
-    'laboratory': laboratory
-}
-
-//Haven't gotten here, decided to cut the story at your rival chooses his pokemon, for the time being.
-async function battleState() {
-    console.log(`${rival.name[0]} wants to fight!`);
-    console.log(`${rival.name[0]} sent out ${capitalize(rival.pokemon[0])}`)
-
-}
-
-async function rivalsPick() {
-    console.log(`${rival.name[0]}: I'll take this one, then!`);
-    if (player.pokemon.includes(`bulbasaur`)) {
-        console.log(`${rival.name[0]} received a Charmander!`);
-        rival.pokemon.push('charmander');
-        laboratory.inventory.splice(laboratory.inventory.indexOf('pokeball3'), 1);
-        return start();
-    } else if (player.pokemon.includes('squirtle')) {
-        console.log(`${rival.name[0]} received a Bulbasaur!`);
-        rival.pokemon.push(`bulbasaur`);
-        laboratory.inventory.splice(laboratory.inventory.indexOf('pokeball1'), 1);
-        return start();
-    } else if (player.pokemon.includes('charmander')) {
-        console.log(`${rival.name[0]} received a Squirtle!`);
-        laboratory.inventory.splice(laboratory.inventory.indexOf('pokeball2'), 1)
-        rival.pokemon.push(`squirtle`);
-        return start();
-    }
-}
-
-async function start() {
-    if (rival.pokemon.length > 0) {
-        console.log(`You both got your pokemon. The end!`);
-        process.exit();
-    } else {
-        let input = await ask(`>_`);
-
-        if (input === ``) {
-            console.log(`Please type in a command.`)
-            return start();
-        }
-        sanitize(input);
-        input = input.trim();
-        input = input.split(' ');
-        let command = input[0];
-        let noun = input[input.length - 1];
-        //if input is help it will print a command list
-        if (command === `help` || noun === `help`) {
-            console.log(`Type "go", "move", or "walk" plus "door" or "stairs" to change player location.\nType "take", "pick up" or "withdraw" plus the name of an item to take that item. Other commands are "inspect", "read", "talk", "use", "drop", or "inventory".`);
-            return start();
-        } else if (currentState === laboratory) {
-            if (laboratory.locked === false) {
-                if (input === '1' || noun === '1') {
-                    console.log(bulbasaur);
-                    let input = await ask(`Oak: So! You want the plant Pokemon, Bulbasaur?`);
```

Comment:
Unfortunately, this is preventing the game from completing on the whole;  on 280, you're comparing incompatible data types.  `currentState` will never equal the `laboratory` object, and thus you'll never get to pick your pokemon.  Small fix, but a crucial one in creating a functioning program.